### DISCUSSION TITLE
Is it a chance to see Flex support until the end of the year ?

### DISCUSSION THREAD

**Discussion by YordanYanakiev** [ruffle-rs/ruffle#979] [General]
Dear Ruffle's devs - I wonder if there is a hope to see Flex support until the deadline for Flash this year, since a lot of projects, and industrial pages rely on Flex - we really hope to see it supported.

**Comment by Herschel** [comment]
Unfortunately Flex support will require both thorough AVM2 support and other features we haven't started to implement yet (including RSLs, etc.), so I would guess it will be over a year before Flex content has a chance of running.

You might try CheerpX (https://leaningtech.com/pages/cheerpxflash.html), which is another product that runs the Flash Player in WebAssembly using x86 emulation.
